category_theory/yoneda.lean

```diff
@@ -95,7 +95,7 @@ let H := (functor.hom ((Cᵒᵖ) ⥤ (Type v₁))) in
 def yoneda_lemma : (yoneda_pairing C) ≅ (yoneda_evaluation C) := 
 { hom := 
   { app := λ F x, ulift.up ((x.app F.2) (𝟙 F.2)),
-    naturality' := begin intros X Y f, ext1, ext1, cases f, cases Y, cases X, dsimp at *, simp at *, erw [←functor_to_types.naturality, yoneda.aux_1, functor_to_types.naturality, functor_to_types.map_id] end },
+    naturality' := begin intros X Y f, ext1, ext1, cases f, cases Y, cases X, dsimp at *, simp at *, erw [←functor_to_types.naturality, yoneda.obj_map_id, functor_to_types.naturality, functor_to_types.map_id] end },
```

Comment:
My screen shatters this line to pieces. Please consider adding replacing the space before `erw` with a newline :smiley: